cinema_service/urls.py

```diff
@@ -2,10 +2,14 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The debug toolbar is included in the URL patterns. Ensure this is only enabled in development environments to avoid exposing debug information in production.